spec/light-client/verification/verification_001_published.md

```diff
@@ -517,6 +517,16 @@ func (ls LightStore) Get(height Height) (LightBlock, bool)
     - returns a LightBlock at a given height or false in the second argument if
     the LightStore does not contain the specified LightBlock.
 
+#### **[LCV-FUNC-VALID-LIGHT-BLOCK.1]**
+
+```go
+func (lb LightBlock) ValidateBasic() error
+```
+
+- Expected postcondition
```

Comment:
I think, typically this should be expected precondition, and the postcondition is something like "no error in case the precondition is satisfied"